app/assets/javascripts/game.js

```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
+  function move_piece(url,new_x_position,new_y_position) {
+    $.ajax({
+      type: 'PUT',
+      url: url,
+      dataType: 'json',
+      data: { piece: { x_position: new_x_position, y_position: new_y_position } },
+      error: function() {
+        alert("Invalid Move!!! Try Again")
```

Comment:
Missing semicolon. You should install a javascript linter into your code editor (JSHint, ESHint)

https://atom.io/packages/atom-jshint